roles/ovirt-datacenter-cleanup/tasks/main.yml

```diff
@@ -0,0 +1,128 @@
+# Cleanup VMPools
```

Comment:
I would create a main.yml which will contains:
- name: vm_pools
  include: vmPools.yml
- name: vms
  include: vms.yml
- name: templates
  include: templates.yml
and so on ....

Reply:
Np, I 'll make it like this.